### PR TITLE
Add Minecraft: New Nintendo 3DS Edition

### PR DESCRIPTION
**Pull request by DerProGamer2000**
Adds Minecraft: New Nintendo 3DS Edition.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/minecraft-new-nintendo-3ds-edition/game.dat [hunk 1]
```diff
@@ -0,0 +1,20 @@
+title = "Minecraft: New Nintendo 3DS Edition"
+description = "Minecraft is a game about placing blocks and going on adventures.\nExplore randomly‐generated worlds and build amazing things from the simplest of homes to the grandest of castles. Play in Creative Mode with unlimited resources or mine deep into the world in survival mode, crafting weapons and armor to fend off the dangerous mobs.\n\n-Taken from Nintendo's Official Site"
```
Comment (Schplee) [inline games/minecraft-new-nintendo-3ds-edition/game.dat]: Remove the `\n` from the description.

File: games/minecraft-new-nintendo-3ds-edition/game.dat [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+[[releases]]
+title = "00040000001b8700"
+region = "usa"
+release_date = "2017-09-13"
+
+[[testcases]]
+title = "00040000001b8700"
+compatibility = "5"
+date = "2017-09-17"
+version = "HEAD-0ceaf22"
```
Comment (Schplee) [inline games/minecraft-new-nintendo-3ds-edition/game.dat]: This isn't a Nightly build.

File: games/minecraft-new-nintendo-3ds-edition/game.dat [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+cpu = "Intel Core i7-4790k 4.80Ghz"
```
Comment (Schplee) [inline games/minecraft-new-nintendo-3ds-edition/game.dat]: Remove the `4.80Ghz`

File: games/minecraft-new-nintendo-3ds-edition/game.dat (hunk 1 above)
Comment (Schplee) [reply games/minecraft-new-nintendo-3ds-edition/game.dat]: Remove `\n\n-Taken from Nintendo's Official Site`

**Comment by TheKoopaKingdom** [comment]
@DerProGamer2000 please fix the issues mentioned above, or this PR will be closed.

**Pull request closed without merging**